### PR TITLE
Create post bugfix

### PR DESCRIPTION
**Pull request by mcverter**
Fixes merging of "create-post" branch.  
https://github.com/reach4help/reach4help/commit/a346879d3521849dffe7dee4d4941320388c53cd
Putting an ANTD Checkbox within a ANTD Modal produces an error in the css file

### CODE REVIEW

File: web-client/src/modules/create/containers/CreatePostContainer.tsx [hunk 1]
```diff
@@ -1,336 +1,196 @@
...
+  const [stepNumber, setStepNumber] = useState(0);
+  const moveForwards = () => setStepNumber(stepNumber + 1);
+  const moveBackwards = () => setStepNumber(stepNumber - 1);
+
+  /* PostDetails */
+  const getTypes = () => [t('modules.create.defaults.postDetails.type')];
+  const [postDetails, setPostDetails] = useState<IPostDetails>({
+    title: t('modules.create.defaults.postDetails.title'),
+    body: t('modules.create.defaults.postDetails.body'),
+    type: getTypes()[0],
```
Comment (shayantist) [inline web-client/src/modules/create/containers/CreatePostContainer.tsx]: Looks great to me overall functionality wise, but hardcoding default values might cause people to miss filling them out themselves. Instead probably best to keep these as an example `placeholder` for each `Input` component in `PostDetailsStep.tsx`

File: web-client/src/modules/create/components/PostDetailsStep.tsx [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+            label={t('modules.create.postDetails.titleLabel')}
+            name="title"
+            rules={[
+              {
+                required: true,
+                message: t('newRequest.form.title_error_message'),
+              },
+            ]}
+          >
+            <Input />
```
Comment (shayantist) [inline web-client/src/modules/create/components/PostDetailsStep.tsx]: In reference to my other comment, maybe replace this with: `<Input placeholder="[default/example value]"/>`

**Review by shayantist — CHANGES_REQUESTED** [review]

File: web-client/src/modules/create/containers/CreatePostContainer.tsx (hunk 1 above)
Comment (mcverter) [reply web-client/src/modules/create/containers/CreatePostContainer.tsx]: fixed

File: web-client/src/modules/create/components/PostDetailsStep.tsx (hunk 2 above)
Comment (mcverter) [reply web-client/src/modules/create/components/PostDetailsStep.tsx]: fixed

**Review by shayantist — APPROVED** [review]

**Pull request merged**